docs/sources/articles/dockerfile_best-practices.md

```diff
@@ -335,12 +331,31 @@ fi
 exec "$@"
 ```
 
-That script then gets copied into the container and run via `ENTRYPOINT` on
-container startup:
+> **Note**: This script uses [the `exec` Bash command](http://wiki.bash-hackers.org/commands/builtin/exec)
+> so that the final running application becomes the container's `PID 1` so that
+> it will receive any Unix signals sent to the container. See the
+> [`ENTRYPOINT`](https://docs.docker.com/reference/builder/#ENTRYPOINT) help for more
+> details.
+
+
+That script is copied into the container and run via `ENTRYPOINT` on
+container start:
 
     COPY ./docker-entrypoint.sh /
     ENTRYPOINT ["/docker-entrypoint.sh"]
 
+This script will allow the user to just start Postgres:
+
+    $ docker run postgres
+
+or, add some parameters to the Postgres server:
+
+    $ docker run postgres postres --help
+
+or, to start a totally different tool, such as Bash:
+
```

Comment:
"or, to start" -> "or, start"?

The repeated "or, " overall reads a bit funny. Perhaps;

This script (will) allow(s) the user to just start Postgres (..) add some parameters .. (..) or start a different tool...